timescaledb/page-index/page-index.js

```diff
@@ -1,33 +1,69 @@
-const overviewPageIndex = require('../overview/page-index/page-index')
-const gettingStartedPageIndex = require('../getting-started/page-index/page-index')
-const howtoPageIndex = require('../how-to-guides/page-index/page-index')
-const quickStartPageIndex = require('../quick-start/page-index/page-index')
-const tutorialPageIndex = require('../tutorials/page-index/page-index')
-
-
 module.exports = [
   {
     title: 'TimescaleDB',
     href: 'timescaledb',
+    pageComponents:["featured-cards"],
+    tags: ["timescaledb", "overview", "get started", "how-to's", "contribute"],
+    keywords: ["TimescaleDB","get started","tutorials", "code", "contribute"],
     excerpt:
-      "All you need to know about TimescaleDB architecture, concepts, setup, how-to's, tutorials, and management",
+      "All you need to know about TimescaleDB architecture, concepts, setup, how-to's, tutorials, management, and contribution.",
+    filePath: "index.md",
     children: [
-      ...overviewPageIndex,
```

Comment:
These imports are needed - without them the pages won't be generated under "TimescaleDB"